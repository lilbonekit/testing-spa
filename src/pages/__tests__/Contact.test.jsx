import { screen, render } from '@testing-library/react';

import { Contact } from '../Contact';

describe('About', () => {
  it('should render correctly', () => {
    render(<Contact />);

    expect(screen.getByRole('heading')).toMatchSnapshot();
  });
});
