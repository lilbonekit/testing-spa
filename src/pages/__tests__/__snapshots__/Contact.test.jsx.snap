// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should render correctly 1`] = `
<h1>
  Hello, from Contact page
</h1>
`;
